source/plugin/text/representations/configurate.rst

```diff
@@ -8,23 +8,24 @@ Configuration Format
     ninja.leaping.configurate.loader.ConfigurationLoader
 
 SpongeAPI offers support to serialize text directly to a Configurate configuration file through the use of the
-``TypeToken`` class. :javadoc:`Text` objects are saved using the same node structure as the ``Text``\'s JSON
-representation.
+``TypeToken`` class. :javadoc:`Text` objects are saved using the same node structure as the ``Text``\'s 
+:doc:`json` on the previous page, but in a more flexible format called :doc:`Human-Optimized Config Object Notation 
```

Comment:
Don't include the words "on the previous page", because we cannot assume that the user has visited that page previously.